src/Hl7.Fhir.Specification.Tests/Validation/ProfileAssertionTests.cs

```diff
@@ -103,18 +102,18 @@ public async T.Task InitializationAndResolution()
         [Fact]
         public void NormalElement()
         {
-            var assertion = new ProfileAssertion("Patient.name[0]", resolve);
-            assertion.SetDeclaredType(FHIRAllTypes.HumanName.GetLiteral());
+            var assertion = new ProfileAssertion("Patient.name[0]", ModelInfo.ModelInspector, resolve);
+            assertion.SetDeclaredType(ModelInfo.CanonicalUriForFhirCoreType(FHIRAllTypes.HumanName));
```

Comment:
This is a rather big change (I remember you mentioned it). Did you discover how on !@#$%#$ the original tests used to work with the literal name, and now they need the full canonical?  Is it a change in the interface, or were the old test just bad?

Reply:
I removed the following method in `ProfileAssertion`, because we don't have `FHIRAllTypes` anymore.

```c#
 public void SetDeclaredType(FHIRAllTypes declaredType)
{
    SetDeclaredType(ModelInfo.CanonicalUriForFhirCoreType(declaredType));
}
```

So the only public method `SetDeclaredType` left is the only one that takes a string as an argument. That string expects a canonical string. That behavior was also in the current version of the SDK. 

Line 107 I changed in the previous PRs, the original line was `assertion.SetDeclaredType(FHIRAllTypes.HumanName);`

So no big change, only that we removed the public method `public void SetDeclaredType(FHIRAllTypes declaredType)`.